### PR TITLE
Adds Desktop_Time_Tracker using Bash Scripting

### PR DESCRIPTION
**Pull request by abdallah1097**
This is a bash script that tracks applications activity e.g. Time elapsed.

Missing Feature: 
- Pie Chart: I've been trying to plot a pie chart in bash but couldn't, the draw_pie_chart has two arrays to be plotted (app names, time). If you can help me with that

Suggested Features to be added:
- Load previous readings and initiate the app time with those loaded values

### CODE REVIEW

File: Bash/Desktop_Time_Tracker/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Desktop Time Tracker
+
+## Description
+'time_tracker.sh` script will track your applications activities e.g. Time Elapsed
```
Comment (HarshCasper) [inline Bash/Desktop_Time_Tracker/README.md:4]: It should be ```
`time_tracker.sh` 
``` and not 'time_tracker.sh`

**Review by HarshCasper — APPROVED** [review]
One tiny change. Rest looks good to me 🥳

**Comment by abdallah1097** [comment]
Required change was made :) Would like to get a feedback from you if possible on #614 and #596 regarding any aspects (coding style, skills, var naming, ...) Any tiny feedback would be appreciated :) :)

**Comment by HarshCasper** [comment]
Thanks for these quality PRs @abdallah1097

**Pull request merged**